webapp/src/components/App/Studies/LauncherDialog.tsx

```diff
@@ -83,6 +84,18 @@ function LauncherDialog(props: Props) {
     },
   );
 
+  const { data: launcherTimeLimit } = usePromiseWithSnackbarError(
+    async () => {
+      return await getLauncherTimeLimit();
+    },
+    {
+      errorMessage: t("study.error.launcherTimeLimit"),
+    },
+  );
+
+  const minSeconds = 3600;
```

Comment:
Create outside the component `const SECOND = 3600;` and replace all occurence of `3600` with it